config/routes.rb

```diff
@@ -74,6 +74,10 @@
   devise_for :users,
              controllers: { omniauth_callbacks: 'omniauth_callbacks' },
              sign_out_via: [:post, :delete]
+  
+  if Rails.env.development?
```

Comment:
It could. But the gem is loaded and so having the route shouldn't prevent anything from loading. It's all a matter of how many places we want to check the ENV variable. If we wrap the routes, seems like we'd also want to wrap the entry in the Gemfile for letter_opener_web as well?